src/Microsoft.Health.Fhir.Shared.Web/DevelopmentIdentityProviderRegistrationExtensions.cs

```diff
@@ -171,8 +171,10 @@ private static IReadOnlyCollection<ApiScope> GenerateSmartClinicalScopes()
 
             scopes.Add(new ApiScope("patient/*.*"));
             scopes.Add(new ApiScope("user/*.*"));
+            scopes.Add(new ApiScope("system/*.*"));
```

Comment:
Can we add `system/*.read` here? That's the default scope for Inferno section 7.

![CleanShot 2023-08-18 at 10 12 38@2x](https://github.com/microsoft/fhir-server/assets/753437/b6df078e-9b81-4dab-a7bb-880af9b9d439)

Reply:
Added!